optimizely/project_config.py

```diff
@@ -598,3 +638,52 @@ def get_variation_from_key_by_experiment_id(self, experiment_id, variation_key):
                           variation_key, experiment_id)
 
         return {}
+
+    def get_flag_variation_by_id(self, flag_key, variation_id):
+        """
+        Gets variation by id.
+        variation_id can be a string or in case of forced decisions, it can be an object.
```

Comment:
this is confusing sentence.

Reply:
Remobved. I forgot to remove the line after I made changes to the code.